### PR TITLE
Bump docker to 9.5.0

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
        ca-certificates \
        curl \
     && curl -s https://install.citusdata.com/community/deb.sh | bash \
-    && apt-get install -y postgresql-$PG_MAJOR-citus-9.4.=$CITUS_VERSION \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-9.5.=$CITUS_VERSION \
                           postgresql-$PG_MAJOR-hll=2.14.citus-1 \
                           postgresql-$PG_MAJOR-topn=2.3.0 \
     && apt-get purge -y --auto-remove curl \
```
Comment (onurctirtir) [inline Dockerfile:23]: this breaks docker builds as we don't have pg13 packages for hll & topn yet

Comment (onurctirtir) [reply Dockerfile]: see e9751947058235cd6dc014c17fd70bb2a9e26115

Comment (hanefi) [reply Dockerfile]: Let's just remove these lines for now, until we have PG13 packages for hll and topn

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+### citus-docker v9.5.0.docker (November 12, 2020) ###
```
Comment (hanefi) [inline CHANGELOG.md:1]: Maybe we should add some entries for missing topn and hll packages in this image. Users could run `CREATE EXTENSION hll` previously, and now they cannot do this. I think this should be documented somewhere

**Review by hanefi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (onurctirtir) [reply CHANGELOG.md:1]: addressed c5f01bd

**Comment by onurctirtir** [comment]
Sorry, couldn't wait that slow ci :/ But changelog edit shouldn't break

**Pull request merged**